ankursnh023.md

```diff
@@ -0,0 +1,8 @@
+-> Apart from all the basic features like forms, various resume templates and many more stuffs. 
+   I think that it would be great to have a feature where the user can rate a particular template 
+   so that based on the ratings of the template we could filter out what's hot in the market so 
+   that it would be convinient for the user to select the top rated/popular template easily.
+
```

Comment:
Can you also add basic features that you expect as resume builder service should have?

Reply:
I think most of our teammates have already suggested the basic features that this application should contain. So, I simply suggested those features that could be added.

Reply:
The goal of this exercise is that everyone pulls their thoughts completely.